docs/source/index.rst

```diff
@@ -28,21 +28,29 @@ This document introduces Olive and provides some examples to get you started.
 
    examples
 
+.. toctree::
+   :maxdepth: 1
+   :caption: FEATURES
+   features/model_transformations_and_optimizations
```

Comment:
it needs a space at line 34 and 42. Otherwise sphinx doesn't pick up the section.